pom.xml

```diff
@@ -210,12 +210,7 @@
         <dependency>
             <groupId>org.testcontainers</groupId>
             <artifactId>postgresql</artifactId>
-            <version>1.14.3</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.springframework.kafka</groupId>
-            <artifactId>spring-kafka-test</artifactId>
+            <version>1.15.0-rc2</version>
```

Comment:
Lage oppgave på å uppe denne asap eller? Isåfall kunne vi gjort det for ba-sak også.

Reply:
Dependabot er enablet for ba-mottak. Kan ta ansvaret med å bumpe ba-sak også når det kommer en release